db/re/item_db_usable.yml

```diff
@@ -587,7 +586,7 @@ Body:
     Buy: 1
     Weight: 10
     Script: |
-      itemheal rand(70,90),rand(20,30);
+      itemheal rand(70,90),0;
```

Comment:
It was fine according to aegis data and there is no value in kro description, please revert

Reply:
Thanks !